en/starter/generator/core_concepts.md

```diff
@@ -215,32 +216,32 @@ Boilerplate code for creating a client object for the Kitura-redis API is includ
+#### Object Storage IBM Cloud service
+This service uses the [Object Storage package](https://github.com/ibm-bluemix-mobile-services/bluemix-objectstorage-serversdk-swift.git) to connect to the IBM Cloud Object Storage service.
 
 Object Storage provides an unstructured cloud data store, which allows the application to store and access unstructured data content.
 
 Boilerplate code for creating a client object for the Object Storage API is included inside `Sources/Application/Application.swift` as an `internal` variable available for use anywhere in the `Application` module.
 
 The connection details for this client are loaded by the configuration boilerplate code and are passed to the Object Storage client.
 
-#### AppID Bluemix service
-This service uses [App ID package](https://github.com/ibm-cloud-security/appid-serversdk-swift) to connect to the Bluemix App ID service.
+#### AppID IBM Cloud service
+This service uses [App ID package](https://github.com/ibm-cloud-security/appid-serversdk-swift) to connect to the IBM Cloud App ID service.
 
 App ID provides authentication to secure your web applications and back-end systems. In addition App ID supports authentication using social identity providers so that users can login with their existing user accounts, such as Facebook and Google.
 
 Boilerplate code for creating a client object for the App ID API is included inside `Sources/Application/Application.swift` as an `internal` variable available for use anywhere in the `Application` module. Extra routes and logic need to be added to make this a authentication boilerplate work. A working example can be found in the [App ID README](https://github.com/ibm-cloud-security/appid-serversdk-swift/blob/master/README.md#example-usage).
 
 The connection details for this client are loaded by the configuration boilerplate code and are passed to the App ID client.
 
-#### Auto-scaling Bluemix service
-This service uses the [SwiftMetrics package](https://github.com/RuntimeTools/SwiftMetrics) for connecting to the Bluemix Auto-scaling service. You can use this to automatically manage your application capacity when deployed to Bluemix.  You will need to define the Auto-Scaling policy (https://console.ng.bluemix.net/docs/services/Auto-Scaling/index.html) to define the rules used to scale the application.
+#### Auto-scaling IBM Cloud service
+This service uses the [SwiftMetrics package](https://github.com/RuntimeTools/SwiftMetrics) for connecting to the IBM Cloud Auto-scaling service. You can use this to automatically manage your application capacity when deployed to IBM Cloud.  You will need to define the Auto-Scaling policy (https://console.ng.bluemix.net/docs/services/Auto-Scaling/index.html) to define the rules used to scale the application.
 
 The connection details for this client are loaded by the configuration boilerplate code and are passed to the SwiftMetrics auto-scaling client.
 
-### Non-Bluemix services
+### Non-IBM Cloud services

```

Comment:
I think this whole section is not needed, there's no non-cloud mode any more.

Reply:
done